galsim/transform.py

```diff
@@ -20,7 +20,7 @@
 """
 
 import galsim
-import numpy
+import numpy as np
```

Comment:
I think this is the only good argument I've heard for any of these style changes.  It's a fair point to have this be consistent in the code so that we know which one to type.  I prefer `numpy` but both you and Rachel both seem to prefer `np`, so I'm fine with make that a standard throughout GalSim.